lms/envs/common.py

```diff
@@ -773,7 +773,6 @@
 CMS_BASE = 'localhost:8001'
 
 # Site info
-SITE_ID = 1
```

Comment:
@ziafazal If you are removing this setting then this usage should also be removed:
https://github.com/edx/edx-platform/blob/master/lms/djangoapps/course_wiki/views.py#L62